CHANGES/1042.cve

```diff
@@ -0,0 +1 @@
+remove vim per CVE-2021-3872
```

Comment:
Regarding `.cve` we have been using these file extensions per https://pypi.org/project/towncrier/, ci should be updated to fail on this
```
    .feature: Signifying a new feature.
    .bugfix: Signifying a bug fix.
    .doc: Signifying a documentation improvement.
    .removal: Signifying a deprecation or removal of public API.
    .misc: A ticket has been closed, but it is not of interest to users.
```